coalib/parsing/LineParser.py

```diff
@@ -82,6 +82,15 @@ def _parse(self, line):
             line,
             self.comment_separators)
         comment = unescape(comment)
+
```

Comment:
Anyway, I don't really think this is the best way to solve this. How about a simple regex on the line before we split it with `__separate_by_first_occurrence` in line 81? `[^ ]#[^ ]` will catch all possible issue :)